src/univalue/.travis.yml

```diff
@@ -26,6 +25,7 @@ addons:
       - pkg-config
 
 before_script:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew uninstall libtool; brew install libtool; fi
```

Comment:
The install will install a newer/different version of libtool which doesn't have an issue that prevented compiling the univalue project on macosx. I'd expect it to be a temporary workaround.
Has this been reported meta-upstream to Travis?